File: cli/generate.go
package cli

import (
	"fmt"

	"devely.com/mob/utils"
)

func Generate() {
	utils.ClearTerminal()
	utils.PrintEmptyLine()
	fmt.Println("📦 Generating...")
	utils.PrintEmptyLine()
	fmt.Println("🚀 Running flutter pub get...")
	utils.RunCommandWithArgs("flutter", []string{"pub", "get"})
	utils.PrintEmptyLine()
	fmt.Println("🚀 Generating assets...")
	utils.RunCommandWithArgs("fluttergen", []string{"."})
	utils.PrintEmptyLine()
	fmt.Println("🚀 Generating i18n...")
	utils.RunCommandWithArgs("flutter", []string{"pub", "run", "fast_i18n"})
	utils.PrintEmptyLine()
	fmt.Println("🚀 Generating xcode project...")
	utils.RunCommandWithArgs("xcodegen", []string{"generate", "--spec", "ios/project.yml", "--project", "ios/"})
	utils.RunCommandWithArgs("pod", []string{"install", "--project-directory=ios"})
}

File: cli/generate.py
from cli.utils import run_shell_command, print_message, show_status

def run():
    print_message("📦 Generating...")

    print_message("🚀 Running flutter pub get...")
    status = run_shell_command("flutter pub get")
    show_status(status)

    print_message("🚀 Generating assets...")
    status = run_shell_command("fluttergen")
    show_status(status)

    print_message("🚀 Generating i18n...")
    status = run_shell_command("flutter pub run fast_i18n")
    show_status(status)

    print_message("🚀 Generating xcode project...")
    status = run_shell_command("cd ios && xcodegen generate && pod install")
    show_status(status)